rider/plugin_description.md

```diff
@@ -7,15 +7,14 @@ in markdown, than to write the xml-encoded html into `plugin.xml`.
 
 ## The Description
 <!-- Plugin description -->
-**Cake for Rider**
+[**Cake for Rider**](https://github.com/cake-build/cake-rider/)
```

Comment:
I would suggest to link to page for extension on cake website here instead of repository

Reply:
good point. (I need to update that PR, too...)

Reply:
@pascalberger I have updated the link